lib/serum/dev_server/command_handler.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Serum.DevServer.CommandHandler do
+  @moduledoc false
+
+  _moduledocp = """
+  Run some commands.
+  """
+
+  @doc "Open the url in a browser."
+  @callback open_url(url :: binary()) :: {:ok | :error}
```

Comment:
`{:ok | :error}` means "`{:ok}` or `{:error}`", not "`:ok` or `:error`."

```suggestion
  @callback open_url(url :: binary()) :: :ok | :error
```